accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,375 @@
+
+NOTE: CaaS Platform stack is also deployed via containers. We will need to update this section when the CaaS Platform team concludes the "add-ons update story".
+
+#### CVE Auditing
+
+By leveraging the existing feature of CVE Auditing, SUSE Manager can detect and notify the user whether the cluster is running a vulnerable image. When SUSE Manager rebuilds the image and patches it, SUSE Manager can also guide the user in modifying the Kubernetes deployment to pull the updated image (in case `imagePullPolicy:` is not `Always`) or do a `kubectl rolling-update`.
+
+##  Future work
+
+### CVE Auditing
+
+A useful addition would be to have SUSE Manager as an admission controller for rejecting pod creation whenever an image used is vulnerable.
+
+### Deploy OpenStack Cloud
+
+If OpenStack Cloud will be deployed using AirShip (still under discussion), the approach described in this RFC can be further reused for deploying and managing an OpenStack Cloud, given that `skuba` is similar to `airshipctl`:
+
+- provision nodes to be part of the cloud
+- tag nodes and bootstrap them to be part of the cloud
+- manage the lifecycle of the cloud
+
+### Run other products in the cluster
+
+It would also be possible, after that the cluster has been deployed, to deploy other SUSE products on top of the cluster (e.g. SUSE OpenStack Cloud, SUSE Enterprise storage): 
+
+- CaaS Platform deploys a SUSE Application Lifecycle operator on top of the cluster with a set of custom resources (CRDs)
+- SUSE Manager can then deploy SOC, SES or any other combination of products on top of the cluster and manage the lifecycle of the deployed products by simply having access to the `kubectl` configuration file
+
+In this case, SUSE Manager would be the central point of deployment for other products based on CaaS Platform and show the health of every cluster installed, while offering the possibility of managing and upgrading the cluster.
+
+### Kubernetes dashboard
+
+In the future developments, SUSE Manager can also embed the Kubernetes dashboard to help the user in deploy applications on top of CaaS Platform.
+The alternative of offering to configure application deployments from SUSE Manager would be seen as a second-choice solution compared to Kubernetes dashboard.
+
+### Re-use of deployment structure
+
+A small part of the deployment of the CaaS Platform can be reused for other Kubernetes distributions (e.g. OpenShift); deploying on bare metal requires machines configured with very specific requirements and that can be solved with AutoYaST (or Kickstart in the Red Hat case).
+
+Every distribution uses then its deployment helper (OpenShift uses `openshift-install`) to bootstrap the cluster. In this regard, `skuba` is only targeting SUSE CaaS Platform and cannot be reused for bootstrapping and managing a cluster that is not CaaS Platform based.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+What if the customer wants to deploy another Kubernetes distribution?
```

Comment:
I think the "generic" part of the RFC answers this question. It would be just a matter of writing `kubectl` versions of most of the most states. In the end, CaaSP and Kubernetes can be two alternative cluster types.

Reply:
I added this comment to the question above.